app/javascript/controllers/apex_nav_controller.js

```diff
@@ -1,81 +1,55 @@
 import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
-  static targets = ["startDate", "endDate", "interval", "uniques", "dropdays", "mainStart", "mainEnd", "tab"]
+  static targets = ["datePreset", "startDate", "endDate", "interval", "uniques", "dropdays", "tab", "card", "mainCard"]
 
-  updateDates(event) {
-    const [startDate, endDate] = JSON.parse(event.target.value)
-    this.mainStartTarget.value = startDate
-    this.mainEndTarget.value = endDate
-
-    this.mainStartTarget.focus()
-    this.mainStartTarget.blur()
-    this.mainEndTarget.focus()
-    this.mainEndTarget.blur()
-    event.target.focus()
-  }
-
-  updateStartDate(event) {
-    this.updateAllTargets(this.startDateTargets, event.target.value)
+  static values = {
+    mainCard: String,
+    agentsCard: String,
   }
 
-  updateEndDate(event) {
-    this.updateAllTargets(this.endDateTargets, event.target.value)
+  connect() {
+    this.tabTargets.forEach((tab) => {
+      if (tab.dataset.card === "main") {
+        if (tab.dataset.tab === this.mainCardValue) {
+          tab.click()
+        }
+      }
+    })
   }
 
-  updateInterval(event) {
```

Comment:
all of these extra methods were only used because when i had multiple forms and i wanted to target the correct inputs. and now they are no longer needed.